test/Microsoft.AspNetCore.Mvc.Core.Test/ApplicationModels/ControllerActionDescriptorProviderTests.cs

```diff
@@ -791,13 +695,14 @@ public void AttributeRouting_ThrowsIfAttributeRoutedAndNonAttributedActions_OnTh
             // Arrange
             var controllerTypeInfo = typeof(UserController).GetTypeInfo();
             var manager = GetApplicationManager(new[] { controllerTypeInfo });
+
             var options = Options.Create(new MvcOptions());
-            options.Value.Conventions.Add(new TestRoutingConvention());
+
```

Comment:
This test broke when I fixed the bug I described here: https://github.com/aspnet/Mvc/pull/8735/files#r234425796

As a result I changed the test setup to be more straightforward (see the convention at the bottom of the file).